src/views/2-Modeling/Modeling.vue

```diff
@@ -1738,13 +1740,13 @@
              this.chartState.alpha = 1;
              this.chartState.aDecay = 0.1;
           }
-          if (response.direction == "up" && this.step === this.step_rmse) {
+          if (response.direction == "up" && this.step <= this.step_rmse+2) {
             this.chartState.strengthy = 1.5;
             this.chartState.strengthy = 2;
             this.chartState.radius = 0;
              this.chartState.strengthr = 0;
-             this.chartState.alpha = .2;
-             this.chartState.aDecay = 0.15;
+             this.chartState.alpha = 1;
+             this.chartState.aDecay = 0.1;
```

Comment:
ok yeah this makes sense to be different on the upscroll. needs more energy to get them to the right place. also looks like the `===` might have been limiting which steps applied to. nice fix!